cmd/createenr_internal_test.go

```diff
@@ -32,3 +36,22 @@ func TestRunCreateEnr(t *testing.T) {
 	err = runCreateEnrCmd(io.Discard, p2p.Config{}, temp)
 	require.NoError(t, err)
 }
+
+func TestCreateENR(t *testing.T) {
+	config := p2p.Config{
+		TCPAddrs: []string{"127.0.0.1:3610"},
+		UDPAddr:  "127.0.0.1:3630",
+	}
+
+	key, err := ecdsa.GenerateKey(crypto.S256(), rand.New(rand.NewSource(time.Now().Unix())))
+	require.NoError(t, err)
+
+	originalENR, err := createENR(key, config)
+	require.NoError(t, err)
+
+	for i := 0; i < 100; i++ {
```

Comment:
there is nothing random to this function, so no need to do 100 times, can just do twice